src/App.tsx

```diff
@@ -25,4 +27,8 @@ function App() {
   );
 }
 
-export default App;
+const mapStateToProps = (store) => ({
+  userName: store.user.name,
+});
```

Comment:
Another option is to use _useSelector_ hook from [react-redux](https://react-redux.js.org/introduction/getting-started) lib.